controllers/login.js

```diff
@@ -213,12 +215,21 @@ router.get('/login/systems/:schoolId', (req, res, next) => {
 });
 
 router.get('/logout/', (req, res, next) => {
-	api(req).del('/authentication') // async, ignore result
-		.catch((err) => { logger.error('error during logout.', { error: err.toString() }); });
-	return authHelper
-		.clearCookie(req, res, { destroySession: true })
-		.then(() => res.redirect('/'))
-		.catch(next);
+	if (res.locals.csrfToken) {
+		api(req).del('/authentication') // async, ignore result
+			.catch((err) => { logger.error('error during logout.', { error: err.toString() }); });
+		authHelper
```

Comment:
I don't think that the function will exit at that point, In this case the function will return null and not the result of the authHelper. If the result of the function is not used why to return null?

Reply:
yeah good point, my mistake

Reply:
done ;)